### PR TITLE
Replace more db.DefaultContext

### PR DESCRIPTION
**Pull request by lunny**
Target #27065

### CODE REVIEW

File: modules/system/db.go [hunk 1]
```diff
@@ -31,5 +32,5 @@ func (f *DBStore) Set(item StateItem) error {
 	if err != nil {
 		return err
 	}
-	return system.SaveAppStateContent(item.Name(), util.BytesToReadOnlyString(b))
+	return system.SaveAppStateContent(db.DefaultContext, item.Name(), util.BytesToReadOnlyString(b))
```
Comment (delvh) [inline modules/system/db.go]: Why can't we change the parameters of `Get`/`Set`?

**Review by delvh — APPROVED** [review]

**Review by 6543 — APPROVED** [review]

File: services/task/migrate.go [hunk 2]
```diff
@@ -63,8 +64,7 @@ func runMigrateTask(ctx context.Context, t *admin_model.Task) (err error) {
 		t.EndTime = timeutil.TimeStampNow()
 		t.Status = structs.TaskStatusFailed
 		t.Message = err.Error()
-		// fixme: Because ctx is canceled here, so the db.DefaultContext is needed.
-		if err := t.UpdateCols(db.DefaultContext, "status", "message", "end_time"); err != nil {
```
Comment (delvh) [inline services/task/migrate.go:67]: oops, this regression should not have occurred.
Thanks to @earl-warren for pointing it out.

Comment (lunny) [reply services/task/migrate.go:67]: What did you mean?

Comment (delvh) [reply services/task/migrate.go:67]: #27740 and #27741

Comment (lunny) [reply services/task/migrate.go:67]: That is not a regression of this PR.

Comment (delvh) [reply services/task/migrate.go:67]: It is, as this PR explicitly replaced it with a cancelable context again

Comment (delvh) [reply services/task/migrate.go:67]: Ah no, sorry.
I misread the code.
You set `ctx = db.DefaultContext` above.

**Pull request merged**